### ISSUE TITLE
how can i retrian the efficientnet model？

### ISSUE THREAD

**Issue by songxh2** [qubvel/efficientnet#22]
please give me a demo, thanks!

**Comment by songxh2** [comment]
finetune on my own datasetes

**Comment by qubvel** [comment]
import keras
from efficientnet import EfficinetNetB0, preprocess_input

# prepare your data
X = ...
y = ...

X = preprocess_input(X)

n_classes = 10

# build model
base_model = EfficinetNetB0(input_shape=(224,224,3), weights='imagenet', include_top=False)
x = keras.layers.GlobalAveragePooling2D()(base_model.output)
output = keras.layers.Dense(n_classes, activation='softmax')(x)
model = keras.models.Model(inputs=[base_model.input], outputs=[output])

# train
model.compile(optimizer='SGD', loss='categorical_crossentropy', metrics=['accuracy'])
model.fit(X, y)

**Comment by songxh2** [comment]
> Hi @songxh2
> 
> ```python
> import keras
> from efficientnet import EfficinetNetB0, preprocess_input
> 
> # prepare your data
> X = ...
> y = ...
> 
> X = preprocess_input(X)
> 
> n_classes = 10
> 
> # build model
> base_model = EfficinetNetB0(input_shape=(224,224,3), weights='imagenet', include_top=False)
> x = keras.layers.GlobalAveragePooling2D()(base_model.output)
> output = keras.layers.Dense(n_classes, activation='softmax')(x)
> model = keras.models.Model(inputs=[base_model.input], outputs=[output])
> 
> # train
> model.compile(optimizer='SGD', loss='categorical_crossentropy', metrics=['accuracy'])
> model.fit(X, y)
> ```

thanks a lot,  good luck for your !

**Comment by aaronll94** [comment]
what format should y follow?

**Comment by LianShuaiLong** [comment]
what format should x and y follow?
